components.md

```diff
@@ -37,7 +37,7 @@ propTypes = {
 ---
 
 
-## Talks <- App
+## ThreadsSection <- App
```

Comment:
Postfix 'Section' is redundant because parent component should contain only one word basically